src/azul/service/hca_response_v5.py

```diff
@@ -23,12 +23,21 @@
 logger = logging.getLogger(__name__)
 
 
+class CompTermObj(JsonObject):
+    """
+    Class defining both a composite and single term object in the FacetObj
+    """
+    term = StringProperty()
+    value = StringProperty()
+    unit = StringProperty()
+
+
 class TermObj(JsonObject):
     """
     Class defining a term object in the FacetObj
     """
     count = IntegerProperty()
-    term = StringProperty()
+    term = CompTermObj()
```

Comment:
@hannes-ucsc I was unable to represent this field in a way that captures both the existing structure of each `TermObject` and the newly added organism age which is a dictionary containing both the value and unit. Could you offer me  some guidance as to how I could go about solving this ambiguity, I disbelieve this present approach is of your liking.
I requested @danielsotirhos feedback on this approach and like me was unable to find a not to time consuming solution that effectively selected one of the two available options for the field. However, he had a very good proposition and I quote "Add a property TermObj.unit and instead of using property ‘value’ always save the value in TermObj.term" such that
```
'termFacets': {
    'organPart': {
        'terms`: [
            {'term': 'lung', 'count': 50},
            {'term': 'heart', 'count' 60}
        ]
    },
    'organismAge': {
        'terms': [
            {'term': 100, 'unit': 'years', 'count' 5},
            {'term': 50, 'unit': 'years', 'count' 10}
        ]
    }
}
```
Preventing us from having to select between property types for term field.

Reply:
Just make `unit` an optional property of TermObj.

Reply:
So, yes, @danielsotirhos' suggestion is smart.